README.md

```diff
@@ -30,42 +30,10 @@ Where `repository url` is a link to an empty repository.
 
 
 ## Recommened setup for the repository
-### Github personal access token
-For pre-commit actions to be able to commit changes to a PR, or for the Cruft update to be able to create a PR with updates from the upstream repo, you need a personal access token (PAT) with the following settings:
 
-**Expiration**: No expiration
-**scopes:**
-* `repo`
-* `workflow`
+To see the recommended setup for the repository, see the following file:
+`{{cookiecutter.project_name}}/.github/recommended_repo_setup.md`
```

Comment:
Do we want this in the readme? That's two lines I'd have to delete every time.

Depends on whether we're targeting only internal use, or also for e.g. potential students. If it's only for us three, I'd remove it. What do you think? :-)

Reply:
I assume you thought it was the created one?

Reply:
Ahh, yes, thanks for clarifying!